easybuild/easyconfigs/d/DBus/DBus-1.10.8-foss-2016a.eb

```diff
@@ -16,7 +16,8 @@ source_urls = ['http://dbus.freedesktop.org/releases/dbus']
 sources = [SOURCELOWER_TAR_GZ]
 
 sanity_check_paths = {
-    'files': ['bin/dbus-%s' % x for x in ['cleanup-sockets', 'daemon', 'launch', 'monitor', 'run-session', 'send', 'uuidgen']] +
+    'files': ['bin/dbus-%s' % x for x in ['cleanup-sockets', 'daemon', 'launch', 'monitor', 'run-session', 'send',
+              'uuidgen']] +
```

Comment:
@migueldiascosta please make this consistent with the other `DBus` easyconfig where you moved `send` down (+1 more DBus easyconfig)